.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Question
+about: I have a question
+title: "[QUESTION] I have a question!"
```

Comment:
Same here, do we need `[QUESTION]` in the title?

Reply:
Ok, fixed.